README.md

```diff
@@ -95,23 +94,22 @@ class TestPoint {
 As we can see, the `jextract` tool generated a `Point2d` class, modelling the C struct, and a `point_h` class which contains static native function wrappers, such as `distance`. If we look inside the generated code for `distance` we can find the following:
 
 ```java
-static final FunctionDescriptor distance$FUNC =
-    FunctionDescriptor.of(Constants$root.C_DOUBLE$LAYOUT,
-                          MemoryLayout.structLayout(
-    	                      Constants$root.C_DOUBLE$LAYOUT.withName("x"),
-                              Constants$root.C_DOUBLE$LAYOUT.withName("y")
-                          ).withName("Point2d"));
-
+static final FunctionDescriptor distance$FUNC = FunctionDescriptor.of(Constants$root.C_DOUBLE$LAYOUT,
```

Comment:
Nothing changed here right? Just indent?

Reply:
Yes. I cut-pasted the new output from latest jextract. May have changed alignment in the process.